### ISSUE TITLE
Pick: EE calculation fails if the object is too close to the corner of the pick region

### ISSUE THREAD

**Issue by ejeschke** [ejeschke/ginga#940]
It seems that the ensquared energy calculation in `Pick` fails with the message:

    (plot_ee) | Error making EE plot: At least one EE function must be provided

if the object is close to the edge of the pick region.

Not sure yet, but I guess this is because the cutout for EE is being done from the cutout for the pick, instead of from the full image.

**Comment by pllim** [comment]
Yes, I only use the cutout image. Wasn't sure if I need to pass the whole image around. I figured when someone wants to measure EE, they would click on the star they want to measure anyway. 🤷

**Comment by ejeschke** [comment]
@pllim, I think this is a simple fix, as I did something similar for the fwhm, IIRC.  I will submit a simple PR for the fix.

**Comment by pllim** [comment]
Sounds good to me. Thank you!
